### PR TITLE
Functional tests - Change selectors for customers pages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Change selectors for customers pages
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/04_customers/*" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18873)

### CODE REVIEW

File: tests/puppeteer/pages/BO/customers/addresses/index.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ module.exports = class Addresses extends BOBasePage {
...
-    this.addressesListTableColumn = `${this.addressesListTableRow} td.column-%COLUMN`;
-    this.addressesListTableColumnAction = this.addressesListTableColumn.replace('%COLUMN', 'actions');
-    this.addressesListTableToggleDropDown = `${this.addressesListTableColumnAction} a[data-toggle='dropdown']`;
-    this.addressesListTableDeleteLink = `${this.addressesListTableColumnAction} a[data-url]`;
-    this.addressesListTableEditLink = `${this.addressesListTableColumnAction} a[href*='edit']`;
+    this.addressesListTableRow = row => `${this.addressesListForm} tbody tr:nth-child(${row})`;
+    this.addressesListTableColumn = (row, column) => `${this.addressesListTableRow(row)} td.column-${column}`;
+    this.addressesListTableColumnAction = row => this.addressesListTableColumn(row, 'actions');
+    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)
+    } a[data-toggle='dropdown']`;
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/customers/addresses/index.js]:
```suggestion
    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)} a[data-toggle='dropdown']`;
```

**Review by NeOMakinG — DISMISSED** [review]
Looks fine

File: tests/puppeteer/pages/BO/customers/addresses/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/customers/addresses/index.js]: I have max length eslint error with this suggestion.

**Review by PierreRambaud — DISMISSED** [review]

File: tests/puppeteer/pages/BO/customers/addresses/index.js (hunk 1 above)
Comment (Progi1984) [reply tests/puppeteer/pages/BO/customers/addresses/index.js]:
```suggestion
    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)} `
    + ` a[data-toggle='dropdown']`;
```
for al lines

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/customers/addresses/index.js]: done thanks

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**